python/lsst/ts/watcher/rules/atcamera_dewar.py

```diff
@@ -0,0 +1,420 @@
+            value_name=info.descr,
+            units=info.units,
+            **level_kwargs,
+        )
+
+    @classmethod
+    def get_schema(cls) -> typing.Dict[str, typing.Any]:
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            description:
+                Configuration for ATCameraDewar
+                You must specify at least one threshold level for each item
+                (min ccd_temp, max ccd_temp, max cold_plate_temp, etc.).
+                It is common to specify a warning level and either a serious
+                or a critical level (not usually both).
+            type: object
+            properties:
+                min_warning_ccd_temp:
+                    description: Minimum CCD temperature (C) below which warning.
+                    type: number
+                    default: -95
+                min_serious_ccd_temp:
+                    description: Minimum CCD temperature (C) below which serious.
+                    type: number
+                min_critical_ccd_temp:
+                    description: Minimum CCD temperature (C) below which critical.
+                    type: number
+                max_warning_ccd_temp:
+                    description: Maximum CCD temperature (C) above which warning.
+                    type: number
+                    default: -93
+                max_serious_ccd_temp:
+                    description: Maximum CCD temperature (C) above which serious.
+                    type: number
+                max_critical_ccd_temp:
+                    description: Maximum CCD temperature (C) above which critical.
+                    type: number
+                max_warning_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which warning.
+                    type: number
+                    default: -110
+                max_serious_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which serious.
+                    type: number
+                max_critical_cold_plate_temp:
+                    description: Maximum cold plate temperature (C) above which critical.
+                    type: number
+                max_warning_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which warning.
+                    type: number
+                    default: -140
+                max_serious_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which serious.
+                    type: number
+                max_critical_cryo_head_temp:
+                    description: Maximum cryo head temperature (C) above which critical.
+                    type: number
+                max_warning_vacuum:
+                    description: Maximum dewar vacuum (Torr) above which warning.
+                    type: number
+                    default: 1.0e-6
```

Comment:
Do you want both a warning level and a serious level? It's totally up to you.

Also: it can easily be changed in the watcher config file, but I'd just as soon make the defaults as reasonable as I can.